docs/install-on-aws.md

```diff
@@ -0,0 +1,233 @@
+
+To keep things simple, you are going to run a single consul daemon on the same host as one of the Swarm managers.
+
+To start, copy the following launch command to a text file.
+
+        $ docker run -d -p 8500:8500 —name=consul progrium/consul -server -bootstrap
+
+Then, use SSH to connect to the "manager0 & consul0" instance. At the command line, enter `ifconfig`. From the output, copy the `eth0` IP address from `inet addr`.
+
+In the text file, find and replace `<consul_ip>` with the IP address.
+
+Using SSH, connect to the "manager0 & etc0" instance. Copy the launch command from the text file and paste it into the command line.
+
+Your consul node is up and running, providing your cluster with a discovery backend.
+
+## Create a high-availability Swarm cluster
+
+After creating the discovery backend, you can create the Swarm managers. Here, you are going to create two Swarm managers in a high-availability configuration. The first manager you run becomes the Swarm's *primary manager*. Some documentation still refers to a primary manager as a "master", but that term has been superseded. The second manager you run serves as a *replica*. If the primary manager becomes unavailable, the cluster elects the replica as the primary manager.
+
+To create the primary manager in a high-availability Swarm cluster, use the following syntax:
+
+        $ docker run -d -p 4000:4000 swarm manage -H :4000 --replication --advertise <manager0_ip>:4000  consul://<consul_ip>
+
+Because this is particular manager is on the same "manager0 & consul0" instance as the consul node, replace both `<manager0_ip>` and `<consul_ip>` with the same IP address. For example:
+
+        $ docker run -d -p 4000:4000 swarm manage -H :4000 --replication --advertise 172.30.0.161:4000  consul://172.30.0.161:8500
+
+Enter `docker ps`. From the output, verify that both a swarm and an consul container are running. Then, disconnect from the "manager0 & consul0" instance.
+
+Connect to the "manager1" instance and use `ifconfig` to get its IP address. Then, enter the following command, replacing `<manager1_ip>`. For example:
+
+        $ docker run -d swarm manage -H :4000 --replication --advertise <manager1_ip>:4000  consul://172.30.0.161:8500
+
+Enter `docker ps` and, from the output, verify that a swarm container is running.
+
+Now, connect to each of the "node0" and "node1" instances, get their IP addresses, and run a Swarm node on each one using the following syntax:
+
+        $ docker run -d swarm join --advertise=<node_ip>:2375 consul://<consul_ip>:8500
+
+For example:
+
+        $ docker run -d swarm join --advertise=172.30.0.69:2375 consul://172.30.0.161:8500
+
+
+Your small Swarm cluster is up and running on multiple hosts, providing you with a high-availability virtual Docker Engine. To increase its reliability and capacity, you can add more Swarm managers and nodes.
```

Comment:
Updated the subsection on consul. Provided a link to instructions for creating a high-availability trio.